app/controllers/runs_controller.rb

```diff
@@ -26,6 +26,10 @@ def show
   end
 
   def index
+    return if current_user.nil?
+    return unless current_user.twitch_user_follows_checked_at.nil? \
+      || current_user.twitch_user_follows_checked_at + 1.day < Time.now.utc
+    current_user.delay.sync_twitch_follows!
```

Comment:
I was actually thinking about this as well.  Would you want it in the top level application controller so that any controller can then "request" follower updates?

Reply:
No strong preference, but in general I prefer to keep things low down and move them up only when we actually want to add them to another controller.